demos/http/http_demo_s3_download_multithreaded/http_demo_s3_download_multithreaded.c

```diff
@@ -596,8 +607,9 @@ static bool getS3ObjectFileSize( const HTTPRequestInfo_t * requestInfo,
             queueOpStatus = retrieveHTTPResponse( responseQueue, &responseItem );
         } while ( queueOpStatus == QUEUE_OP_WOULD_BLOCK );
 
-        if( queueOpStatus != QUEUE_OP_SUCCESS )
+        if( ( queueOpStatus != QUEUE_OP_SUCCESS ) || ( responseItem.response.statusCode != 206 ) )
         {
+            LogError( ( "Recieved repsonse with unexpected status code: %d", responseItem.response.statusCode ) );
```

Comment:
If the status code was OK, but the queue op failed I don't think we want to print this message.